configuration/enable-powerBI-data-collection.md

```diff
@@ -0,0 +1,65 @@
+---
+description: This article explains how to enable Power BI data collection for SysKit Point.
+---
+
+# Enable Power BI Data Collection
+
+Enabling Power BI data collection for SysKit Point requires initial setup through your Microsoft 365 Azure Portal. 
+
+## Create a Power BI Security Group 
+
+ * **Click the Azure Active Directory (1)**; on the Home page of your Azure portal. 
+ * **Select Groups (2)**; located under the Manage section on the left side of the screen. Select Groups
+ * **Click the [New group button (3)](https://portal.azure.com/#view/Microsoft_AAD_IAM/AddGroupBlade)** on the top bar. 
+ 
+![Azure Active Directory - Create New Group](../.gitbook/assets/enable-powerbi-data-collection_aad-newgroup.png)
+
+Fill out the following details when creating the new group:
+
+ * Under **Group Type (1)**, select **Security (2)**
+ * **Name (3)** the group **PowerBI Group** or similar
+ * For the **Group Description (4)** list the purpose of the new group
+ * Set the **toggle as NO** when asked whether Azure AD roles can be assigned to the group
+ * Set the Membership Type as **Assigned (5)**
+ * Select the **Owners and Members (6)** of the Group
+
```

Comment:
tu još fali dodavanje SysKit Point Power Platform app registracije (ono što sam kreirao) u membere ove security grupe
ne bih to stavljao u please note a nema tu objašnjenja u stepovima